src/LasagnaController.h

```diff
@@ -45,13 +45,13 @@ class LasagnaController
   typedef struct __attribute__((packed)) {
     float gain                 =   0.14/1000;   // Total gain magnititude (g / m)
     float damping              =   1.2;         // damping ratio (unitless)
-    float v_gain               =   0;           // velocity gain (g/s / m/s) 
-    float h_gain               =   0;           // altitude gain (m/s / km)
+    float v_gain_override      =   0;           // velocity gain (g/s / m/s) 
+    float h_gain_override      =   0;           // altitude gain (m/s / km)
     float bal_dldt             =   0.666;       // balast dl/dt (g / s)
-    float val_dldt_a           =   0;           
-    float val_dldt_b           =   3;           // valve dl/dt (g / s))
-    float bal_tmin             =   2;           // minimum ballast event time (s)
-    float val_tmin             =   3;           // minimum valve event time (s)
+    float val_dldt_slope       =   0;           
```

Comment:
Does this have a unit?

Reply:
uhh it might be pascals to grams, but I forget specs of overpressure and would rather leave it blank than look it up again or comment something wrong.